### PR TITLE
Refactor layout

### PR DESCRIPTION
**Pull request by bschmoker**
@ikiril01 @gtback 
How does this look? [live here](https://cyboxproject-github-io-dstar.c9.io/_out/)

cc: @stixproject

### CODE REVIEW

**Comment by ikiril01** [comment]
@bschmoker looks good! There's an extra '.' in the href and tag for the cybox@mitre.org address; otherwise, feel free to merge.

**Comment by bschmoker** [comment]
oops, `sed` typo - amended

Added the logo in 54ad283 - lmk if that's cool (same live link as above)

File: _includes/nav.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="{{ site.root }}/">{{ site.name }}</a>
+      <a href="https://cyboxproject.github.io" class="pull-left"><img src="/images/logo.png" style="max-height: 100px; max-width: 120px; margin-top: 3px;"></a>
```
Comment (gtback) [inline _includes/nav.html]: can we leave {{site.root}} here?

File: index.md [hunk 2]
```diff
@@ -40,33 +40,6 @@ no_in_page_title: true
 <p class="lead text-center">
   Have questions, comments, or feedback? Want to set up a teleconference or in-person meeting?
   <br/>
-  <strong>Reach out to us at <a href="mailto:stix-taxii@hq.dhs.gov">stix-taxii@hq.dhs.gov</a>!</strong>
+  <strong>Reach out to us at <a href="mailto:cybox@mitre.org.">cybox@mitre.org</a>!</strong>
```
Comment (gtback) [inline index.md]: Why was this changed (other than the awkwardness of contacting "stix-taxii" about CybOX)? I'm assuming this will change with the Oasis transition as well?

**Comment by gtback** [comment]
A couple comments, but otherwise :+1:

File: _includes/nav.html (hunk 1 above)
Comment (bschmoker) [inline _includes/nav.html]: unfortunately `site.root`doesn't resolve to the base URL (just an empty string), so it's basically just a no-op

File: index.md (hunk 2 above)
Comment (bschmoker) [inline index.md]: :hamburger:

Comment (gtback) [inline index.md]: :banana:
